ql/src/test/results/clientpositive/llap/udf_greatest.q.out

```diff
@@ -63,7 +63,7 @@ STAGE PLANS:
           alias: src
           Row Limit Per Split: 1
           Select Operator
-            expressions: 'c' (type: string), 'a' (type: string), 'AaA' (type: string), 'AAA' (type: string), '13' (type: string), '2' (type: string), '03' (type: string), '1' (type: string), null (type: double), null (type: double), null (type: double), null (type: double), null (type: double), null (type: double)
```

Comment:
Why did these types change? Seems unrelated to this patch. Was it an existing bug?

Reply:
There is a small change in the patch which updates the type inference rule for void/null. Prior to the change the expressions were being inferred as `Double` in this case. With the change it is appropriately inferred as `String` (since rest of the expressions within this UDF (`GREATEST('a', 'b', null )`) is interpreted as string.